tests/introspection/iOS/iOSApiCtorInitTest.cs

```diff
@@ -220,6 +220,8 @@ protected override bool Skip (Type type)
 			case "INGetAvailableRestaurantReservationBookingsIntentResponse": // Objective-C exception thrown.  Name: NSInternalInconsistencyException Reason: Unable to initialize 'INGetAvailableRestaurantReservationBookingsIntentResponse'. Please make sure that your intent definition file is valid.
 			case "INGetRestaurantGuestIntentResponse": // Objective-C exception thrown.  Name: NSInternalInconsistencyException Reason: Unable to initialize 'INGetRestaurantGuestIntentResponse'. Please make sure that your intent definition file is valid.
 				return TestRuntime.CheckXcodeVersion (10,0);
+			case "RPBroadcastPickerView": // Symbol not available in simulator
```

Comment:
:(

Please file a rdar and add this to the checklist of things to audit before releasing xcode10 support

Reply:
Filled as radar://41135211